src/Reflection/ReflectionClass.php

```diff
@@ -491,6 +495,44 @@ public function hasConstant(string $name) : bool
     }
 
     /**
+     * Get the reflection object of the specified class constant.
+     *
+     * Returns null if not specified.
+     *
+     * @param string $name
+     * @return ReflectionClassConstant|null
+     */
+    public function getReflectionConstant(string $name) : ?ReflectionClassConstant
+    {
+        return $this->getReflectionConstants()[$name] ?? null;
+    }
+
+    /**
+     * Get an array reflection object of the defined constants in this class.
+     *
+     * @return ReflectionClassConstant[]
+     */
+    public function getReflectionConstants() : array
```

Comment:
Note to self: this block is to be changed at merge time (not a reduce operation)